test/integration/sign_in_test.rb

```diff
@@ -283,6 +283,18 @@ class SignInTest < SystemTest
     assert page.has_content? "Your account was blocked by rubygems team. Please email support@rubygems.org to recover your account."
   end
 
+  test "sign in to deleted account" do
```

Comment:
Should check forgotten password email token login too. It's probably fine, but since it's an alternate login method, I think we should have coverage.

Reply:
added to the other test file, good call